workflows/phylorthology.nf

```diff
@@ -97,280 +85,204 @@ def create_og_channel(LinkedHashMap row) {
         meta.num_grps = row.num_tax_grps
     // add path(s) of the OG file to the meta map
     def og_meta = []
-        og_meta = [ meta, [ file(row.file) ] ] 
+        og_meta = [ meta, [ file(row.file) ] ]
     return og_meta
 }
 
 workflow PHYLORTHOLOGY {
-    
     ch_inflation = ch_mcl_inflation.toList().flatten()
     ch_versions = Channel.empty()
 
     //
     // SUBWORKFLOW: Read in samplesheet, validate and stage input files
     //
-    // First the full set
-    ch_all_data = INPUT_CHECK (
-        ch_input,
-        ch_data_dir
-    )
+    ch_all_data = INPUT_CHECK(ch_input)
     ch_versions = ch_versions.mix(INPUT_CHECK.out.versions)
+    complete_prots_list = ch_all_data.complete_prots.collect { it[1] }
+    mcl_test_prots_list = ch_all_data.mcl_test_prots.collect { it[1] }
 
-    // Pull out the test and full sets
```

Comment:
SO happy to have all of this gone - all originated from a hacky solution to get busco to work at all before I had any clue what I was doing.